embed-in-chrome-extension/server.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const express = require("express");
+const axios = require("axios");
+const app = express();
+
+app.get("/script.js", (req, res) => {
+  res.sendFile(path.join(__dirname, "script.js"));
+});
+
+app.get("/hb.js", (req, res) => {
+  res.sendFile(
```

Comment:
Suppose somebody has a golang server; do you recommend they use NPM for retrieving the source?

Reply:
not a blocker here just curious what you would do if you were doing CSP and had to serve this from the backend

Reply:
alternatively, we could add unpkg to the allowed hosts and import that way. But i'm demonstrating the minimum needed csp to get hyperbeam working. Also it's a very bad idea to whitelist unpkg because it defeats the purpose of csp by basically allowing arbitrary code execution from random fly-by-night npm packages

Reply:
in reality, the hyperbeam would be bundled into the client bundle without needing a separate file being served. Also I avoided some bundler like vite or parcel because their dev servers are not CSP.

Reply:
yeah makes sense. i'm guessing you would download the file and have nginx serve it if this was for prod?

Reply:
man i don't think github's comments are realtime

Reply:
:( starting to miss MR reviews on gitlab already

Reply:
test comment